kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/util/Notifier.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+package org.kitodo.mediaserver.core.util;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Date;
+import java.util.Locale;
+
+import org.kitodo.mediaserver.core.api.INotifier;
+import org.kitodo.mediaserver.core.config.NotifierProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * Simple notifier.
+ */
+@Component
+public class Notifier implements INotifier {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Notifier.class);
+
+    private JavaMailSender mailSender;
+    private NotifierProperties notifierProperties;
+
+    @Autowired
+    public void setNotifierProperties(NotifierProperties notifierProperties) {
+        this.notifierProperties = notifierProperties;
+    }
+
+    @Autowired
+    public void setMailSender(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    /**
+     * Appends a message to a notification, including timestamp.
+     *
+     * @param buffer The notification buffer
+     * @param message the message
+     * @param includeTimestamp Whether to include a timestamp with each message
+     */
+    @Override
+    public void add(StringBuffer buffer, String message, boolean includeTimestamp) {
```

Comment:
You are not using the boolean param here. With this solution, the method should rather be called addWithTimestamp and have no includeTimestamp param.